packages/renderer/src/lib/troubleshooting/TroubleshootingPageStores.svelte

```diff
@@ -25,7 +25,7 @@ onDestroy(() => {
 });
 </script>
 
-<div class="flex flex-col bg-charcoal-600 p-4 rounded-lg">
+<div class="flex w-full h-fit m-4 mr-8 flex-col bg-charcoal-600 p-4 rounded-lg">
```

Comment:
Remove mr-8, it doesn't match the other pages and gives the stores an abnormally large right border.

Reply:
if you have the scrollbar it's hiding the text below